validator/remote/src/integration-test/resources/responses/get_aggregate_attestation_responses/getAggregateAttestationELECTRA.json

```diff
@@ -0,0 +1,21 @@
+{
+  "version": "phase0",
```

Comment:
why `phase0` here?

Reply:
Well this required some refactoring: Basically the milestone was ignored from the response (since the old api doesn't need it...).
I did the required changes which are basically returning an `ObjectAndMetaData<Attestation>` instead of `Attestation`. The additional meta data (basically the header) is simply ignored when not needed (the v1 api for example).
The test now checks the milestone

Reply:
actually the milestone was ignored from the response, did some refactoring to check it too